3p/3d-gltf/index.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable import/no-deprecated */
```

Comment:
Would it make sense to convert these file-level disable directives to line-level directives? That way, additional deprecated imports cannot inadvertently sneak in with future changes, and developers will have to be explicit about them.

Reply:
It's unfortunately not just the import that triggers it; every usage of this deprecated imported function triggers the alert (which is something like 10 times in at least one of these files). There are only a 3 files iirc that violate it and they are tucked away in `3p/`, so this felt like a reasonable use of file-level disabling, while ensuring any new code is held to a strict standard

Reply:
Agree, thanks for the context.